sessionctx/variable/session.go

```diff
@@ -1492,6 +1497,66 @@ func (s *SessionVars) AllocNewPlanID() int {
 	return s.PlanID
 }
 
+// Clear earliest CacheResult.
+func (s *SessionVars) clearEarliestCacheResult() {
+	var key uint32
+	minTime := time.Now()
+	for k, v := range s.cacheRes {
+		if v.LastUpdateTime.Before(minTime) {
+			minTime = v.LastUpdateTime
+			key = k
```

Comment:
This gives the 'random' expired cache item rather than the 'earliest' one.

But I'm not sure if this is a problem for our 'performance benchmark' test...maybe it's just fine.

Reply:
done